modules/installation-creating-azure-bootstrap.adoc

```diff
@@ -12,12 +11,8 @@ ifeval::["{context}" == "installing-azure-stack-hub-user-infra"]
 :ash:
 :cp: Azure Stack Hub
 endif::[]
-ifeval::["{context}" == "installing-restricted-networks-azure-user-provisioned"]
-:azure:
-:cp: Azure
-endif::[]
 
-:_mod-docs-content-type: PROCEDURE
+:_content-type: PROCEDURE
```

Comment:
The meta data should not be updated. `_mod-docs-content-type: PROCEDURE` is the correct meta data.